taskcluster/app_services_taskgraph/build_config.py

```diff
@@ -10,20 +10,45 @@
 from taskgraph.util.memoize import memoize
 
 
+EXTENSIONS = {
```

Comment:
This is partially what AC implements - https://github.com/mozilla-mobile/android-components/blob/8da8237bf9ccf1cf43088b995db5d4ca02c28a69/taskcluster/ac_taskgraph/build_config.py#L37 with some adjustments below.